### PR TITLE
fixed the way rotest is launching django server

### PR DESCRIPTION
**Pull request by IamShobe**

### CODE REVIEW

File: tests/cli/test_server.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import sys
 
 import mock
+import os
```
Comment (osherdp) [inline tests/cli/test_server.py]: Import order

File: src/rotest/cli/server.py [hunk 2]
```diff
@@ -37,8 +38,11 @@ def start_server(server_port, run_django_server, django_port):
     try:
         if run_django_server:
             print("Running the Django server as well")
+            app_directory = os.path.dirname(search_config_file())
+            manage_py_location = os.path.join(app_directory, "manage.py")
             django_process = subprocess.Popen(
-                ["django-admin",
+                ["python",
```
Comment (osherdp) [inline src/rotest/cli/server.py]: Works well for windows?

Comment (IamShobe) [reply src/rotest/cli/server.py]: Yes.
I actually worked on a Windows PC,
Also tox passed in Windows (except for the import order)

File: tests/cli/test_server.py (hunk 1 above)
Comment (IamShobe) [reply tests/cli/test_server.py]: Done

**Pull request merged**